oauth2client/contrib/django_util/__init__.py

```diff
@@ -160,19 +175,64 @@ def drive_required(request):
    from oauth2client.contrib.django_util.signals import oauth2_authorized
 
    def test_callback(sender, request, credentials, **kwargs):
-       print "Authorization Signal Received %s" % credentials.id_token['email']
+       print(
+       'Authorization Signal Received {0}'.format(
+           credentials.id_token['email']))
 
    oauth2_authorized.connect(test_callback)
 
+Adding Credentials To An Existing Django User System
+=====================================================
+
+As an alternative to storing the credentials in the session, the helper
+can be configured to store the fields on a Django model. This might be useful
+if you need to use the credentials outside the context of a user request. It
+also prevents the need for a logged in user to repeat the OAuth flow when
+starting a new session.
+
+To use, change ``settings.py``
+
+.. code-block:: python
+   :caption:  settings.py
+   :name: storage_model_config
+
+   GOOGLE_OAUTH2_STORAGE_MODEL = {
+    'model': 'path.to.model.MyModel',
+    'user_property': 'user_id',
+    'credentials_property': 'credential'
+    }
+
+Where ``path.to.model`` class is the fully qualified name of a
+``django.db.model`` class containing a ``django.contrib.auth.models.User``
+field with the name specified by `user_property` and a
+:class:`oauth2client.contrib.django_util.models.CredentialsField` with the name
+specified by `credentials_property`. For the sample configuration given,
+our model would look like
+
+.. code-block:: python
+   :caption: models.py
+   :name: storage_model_model
+
+   from django.contrib.auth.models import User
+   from oauth2client.contrib.django_util.models import CredentialsField
+
+   class MyModel(models.Model):
+        #  ... other fields here ...
+       user = models.OneToOneField(User)
+       credential = CredentialsField()
+
 """
 
+from importlib import import_module
```

Comment:
[Import only modules, not classes, functions, or other elements of modules](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports).

Yes, much of the existing code wildly violates this but why dig the hole deeper?